db/cassandra/src/main/java/org/commonjava/indy/cassandra/data/CassandraStoreDataManager.java

```diff
@@ -266,6 +266,12 @@ public Set<Group> affectedBy( Collection<StoreKey> keys ) throws IndyDataExcepti
                         {
                             ArtifactStore store = getArtifactStoreInternal( gKey );
 
+                            if ( store == null )
+                            {
+                                processed.add( gKey );
```

Comment:
Can we have a log here to see which store is null as affected? I think this can help to find potential problem.

Reply:
Done, i was thinking to use yours. But no worry, let's keep this. : )